collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
+
+mkdir -p ${NETWORK_LOG_PATH}/
+
+NODES="${NETWORK_NODES:-$(oc get nodes --no-headers -o custom-columns=':metadata.name')}"
```

Comment:
NETWORK_NODES needs to be a bash argument (I don't think we can read in environment variables; like this).